api/src/pcapi/alembic/versions/20230522T100913_dcfd7657a244_add_educational_redactor_column_to_collective_offer_request.py

```diff
@@ -0,0 +1,30 @@
+"""add-educational-redactor-column-to-collective-offer-request
```

Comment:
Ce commentaire est affiché quand on lance par exemple `alembic history`. Il est donc pratique de le rendre lisible  

```suggestion
"""Add educationalRedactorId to collective_offer_request table
```